test/apps/encoding/src/routes/fünke.json.js

```diff
@@ -4,6 +4,6 @@ export function get(req, res) {
 	});
 
 	res.end(JSON.stringify(
-		"I'm afraid I just blue myself"
+		'I\'m afraid I just blue myself'
```

Comment:
This change shouldn't be necessary if you're using 5.2.0 of the lint config. See https://github.com/sveltejs/eslint-config/pull/8

Reply:
i was using the old version. rolling this back.